closure-deps/bin/closuremakedeps.js

```diff
@@ -215,7 +226,7 @@ async function main(opt_args) {
 
   const sources = new Set((args.file || []).map(resolve));
   const roots = (args.root || []).map(resolve);
-  const excluded = new Set((args.exclude || []).map(resolve));
+  const excluded = (args.exclude || []).map(resolve);
```

Comment:
This tries to resolve a string as a path relative to the cwd - this doesn't make sense if these are now globs?